reference/service/pulumi-button.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Creating a 'Deploy with Pulumi' Button
+---
+
+The 'Deploy with Pulumi' button provides a way to easily create new Pulumi projects within a web browser. The button can be embedded in README files in GitHub repositories/Gists, blog posts, or other web pages.
+
+For example, the following button can be clicked to configure and create a new empty JavaScript project:
+
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new?template=https://github.com/pulumi/templates/javascript)
+
+This document describes how you can create Pulumi buttons for your own Pulumi templates, examples, and apps.
```

Comment:
It might help to explicitly call out that there are two steps to creating a button:
1. Include template metadata in your `Pulumi.yaml`
2. Create a button URL

Just to guide through the next two `##` headers and make clear both are required.